crates/swc_cli/Cargo.toml

```diff
@@ -20,17 +28,18 @@ rayon = "1"
 relative-path = "1.6.1"
 serde = {version = "1", features = ["derive"]}
 serde_json = {version = "1", features = ["unbounded_depth"]}
-swc = {version = "0.152.0", path = "../swc"}
+swc = {version = "0.152.0", path = "../swc", features = ["plugin"]}
```

Comment:
`swc/plugin` should be optionally enabled.

Reply:
Cli should always enable plugin - are you saying it should be built same as wasm does by manually opt into features? Isn't it ok to rely on wasmer/sys as we already did?

Reply:
No, 

> Cli should always enable plugin

This is simply not possible, because `cranelift` does not support some platforms. We have to build cli for those platforms anyway.

Reply:
How do we build `/node` currently for those platforms? Does it mean current `@swc/core` doesn't have plugin for those platforms?

Reply:
Yes, those platform lacks plugin support.

Reply:
ugh, what a headache. Updated default to not to enable plugin by default (https://github.com/swc-project/swc/pull/4035/files#diff-f3d6e8402f7bbe6d2b9d1ce9ca08cc1b4c8406bdae6dc77414e22d65b61a601eR16). 

May need CI side update to enabling this, probabaly with separate PR as current one's not easy to swallow already.